behat/symfony2-extension/2.1/behat.yaml.dist

```diff
@@ -0,0 +1,7 @@
+default:
+    extensions:
+        Behat\Symfony2Extension:
+            kernel:
+                bootstrap: vendor/autoload.php
```

Comment:
I agree that behat loads it, but extension tries to load file from default configuration (`app/autoload.php`) and throws ProcessingException (`Could not load bootstrap file. Please check your configuration at "kernel.bootstrap"`)

I know the purpouse of this option and I prefer to not do such workarounds but if the recipe is to pre-configure this is needed.